actions/ack_incident.py

```diff
@@ -2,13 +2,20 @@
 
 
 class AckIncident(PagerDutyAction):
-    def run(self, event_keys):
+    def run(self, **param_dict):
```

Comment:
Getting formats right for passing in a dict via WebUI or CLI can be a bit of PITA. I wonder if we should do something like have two separate parameters, one for an array of incident_keys, and one for an email. Could also allow setting a default email address for acknowledgement?

(Same comment applies for `resolve_incident`